docs/admin/troubleshooting/cluster-health-check.md

```diff
@@ -1,8 +1,82 @@
+
+This may reflect a networking issue from or to your Kubernetes cluster regardless of Run:ai components. In some cases, it may indicate an issue with one or more Run:ai services that communicate with the Control Plane, which are:
+* Cluster sync (`cluster-sync`)
+* Agent (`runai-agent`)
+* Asset sync (`asset-sync`)
+
+### Troubleshooting actions:
+
+* First, check that the Run:ai services that communicate with the Control Plane are up and running.
+
+  Run:
+
+  `kubectl get pods -n runai | grep -E 'runai-agent|cluster-sync|assets-sync'`
+
+* Verify Run:ai services logs. Inspecting the logs of the Run:ai services that communicate with the CP is an essential first step to identify any error messages or connection issues.
+
+  Run the following command on each one of the services:
+
+  ```
+  kubectl logs runai-agent -n runai
+  kubectl logs cluster-sync -n runai
+  kubectl logs assets-sync -n runai
+  ```
+
+* Check the network connection from the runai namespace in your cluster to the Control Plane. You can do that by running a connectivity check pod.
+
+  Create a pod within the runai namespace. This pod can be a simple container with basic network troubleshooting tools, such as curl or get. Use the following command to determine if the pod can establish connections to the necessary Control Plane endpoints:
+
+  `kubectl run control-plane-connectivity-check -n runai --image=wbitt/network-multitool --command -- /bin/sh -c 'curl -sSf <control-plane-endpoint> > /dev/null && echo "Connection Successful" || echo "Failed connecting to the Control Plane"'`
+
+  Replace `control-plane-endpoint` with the URL of the Control Plane in your environment.
+
+* Check potential network issues. You can use the following guidelines:
+  
+    * Ensure that the network policies in your Kubernetes cluster allow communication between the Run:ai services that communicate with the Control Plane to the Control Plane.
+   
+    * Check both Kubernetes Network Policies and any network-related configurations at the infrastructure level.
+  
+    * Verify that the required ports and protocols are not blocked. 
+
+* Contact Run:ai Support
+
+  If the issue persists and you couldn’t resolve it after completing the above steps, contact Run:ai support for assistance.
+
+!!! Note 
+    The above steps can also be relevant if you installed the cluster and it is stuck in the status “Waiting to connect” for a long time.
+
+## Cluster has Service issues
+
+When a cluster is in the status “Service issues”, this means that one or more Run:ai services that are running in the cluster are not available.
+
+* In this case, first run the following command, to verify which are the non-functioning services, and get more details about deployment issues and resources required by these services that may not be ready (e.g. ingress is was not created or is unhealthy): 
```

Comment:
@gal-revach Same as before. If this is a start of a procedure, then use a numbered list.
```suggestion
* First run the following command, to verify which are the non-functioning services, and get more details about deployment issues and resources required by these services that may not be ready (for example, ingress was not created or is unhealthy): 
```